### ISSUE TITLE
Add list box question type

### ISSUE THREAD

**Issue by s8321414** [nextcloud/forms#1340]
**Nextcloud (please complete the following information):**
- Nextcloud-Version: 24.0.4
- Forms-Version: 2.5.1

**Is your feature request related to a problem? Please describe.**
When we use dropdown question type, sometimes we may need to choose multiple answers.

**Describe the solution you'd like**
Implement list box question type in Forms.

**Comment by Chartman123** [comment]
By now, the dropdown questions allow only one option to be selected. If you need to choose multiple answers, you have to use the checkbox question type for now.

**Comment by s8321414** [comment]
> By now, the dropdown questions allow only one option to be selected. If you need to choose multiple answers, you have to use the checkbox question type for now.

I see, but we can give user more options to design their forms if we have this feature :smiley:

**Comment by Chartman123** [comment]
Yes, that's why I didn't close the issue but changed it to "to develop". However, we probably won't have it in the next release but in the one after :)
